lib/pf/Switch/Ubiquiti/Unifi.pm

```diff
@@ -292,11 +310,8 @@ sub _deauthenticateMacWithHTTP {
             }
         }
     }
-    $logger->info("Deauth on $count access points");
-
-    unless($response->is_success) {
-        $logger->error("Can't send request on the Unifi controller: ".$response->status_line);
-        return;
+    if($count > 0) {
+    	$logger->info("Deauth on $count access points");
     }
 
     $logger->info("Switched status on the Unifi controller using command $command");
```

Comment:
This sentence could be wrong if your device has not been found and your access points are defined by AP in PacketFence config.

Reply:
This is still a problem for me.

Reply:
I misunderstand logic. In fact, this sentence will never appear in logs if AP are defined by IP addresses.